src/test/mocks/vaults.ts

```diff
@@ -1,4 +1,4 @@
-import { bnOrZero } from '@shapeshiftoss/chain-adapters'
+import { bn } from '@shapeshiftoss/chain-adapters'
```

Comment:
While at it, can we also import this from `lib/bignumber`?
```suggestion
import { bn } from 'lib/bignumber/bignumber'
```

That's another issue that we have, where bn gets imported automatically by editors from the first TypeScript Language Server option, while the `lib/bignumber` one is actually further in the list

<img width="843" alt="image" src="https://user-images.githubusercontent.com/17035424/168053995-da2536b6-6c61-4e7e-a6d8-a4ef12b5ccef.png">
<img width="747" alt="image" src="https://user-images.githubusercontent.com/17035424/168053881-1be49ee6-70f4-416b-968a-ac196fb121e7.png">

Reply:
Or well, might as well keep this PR atomic:
https://github.com/shapeshift/web/pull/1760

Reply:
Hmm, why do we explicitly export `bn` from chain adapters if we don't want to use it?

Reply:
Agree with your other PR though - I don't know why it was done this way.

We should just merge yours 👌

Reply:
> Hmm, why do we explicitly export `bn` from chain adapters if we don't want to use it?

It's not by design, it's just that in lib, we wanted to consolidate all utils in one `utils` package that re-exports them all for easy import but by doing that, we effectively make them available for consumers. https://github.com/shapeshift/lib/pull/641 tackles it